src/dispatchers/mouseDispatcher.ts

```diff
@@ -2,110 +2,119 @@
 
 module Plottable {
 export module Dispatcher {
-  export class Mouse extends Dispatcher.AbstractDispatcher {
-    private _mouseover: (location: Point) => any;
-    private _mousemove: (location: Point) => any;
-    private _mouseout: (location: Point) => any;
+  export class Mouse {
+    private static dispatcherKey = "__Plottable_Dispatcher_Mouse";
+    private _svg: SVGElement;
+    private _measureRect: SVGElement;
+    private _lastMousePosition: Point;
+    public broadcaster: Core.Broadcaster;
 
     /**
-     * Constructs a Mouse Dispatcher with the specified target.
+     * Get a Dispatcher.Mouse for the <svg> containing elem. If one already exists
+     * on that <svg>, it will be returned; otherwise, a new one will be created.
```

Comment:
We could create a new `Dispatcher.Mouse` every time, but since they're all doing the same calculation (relative to the `<svg>` they're attached to), that would be redundant.